src/shared/common/components/workstack.jsx

```diff
@@ -69,7 +69,12 @@ class WorkstackAllocate extends Component {
     }
 
     doesFilterMatchData(filter, row, column) {
-        const value = this.getCellValue(row, column);
+        let value = this.getCellValue(row, column);
+
+        if (typeof value === 'object' && value != null) {
```

Comment:
To make this a concrete check for an object use:
```suggestion
        if (value && Object.keys(value).length !== 0 && value.constructor === Object) {
```
as the current implementation would fail if for some reason value was a RegEx. I think in this context this is highly unlikely however.

See https://stackoverflow.com/a/32108184 for more info

Reply:
I have addressed this.